packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,191 @@
+    column-gap: var(--space-2);
+  }
+  .rt-DataListRoot.rt-r-gap-3 {
+    row-gap: var(--space-3);
+    column-gap: var(--space-3);
+  }
+  .rt-DataListRoot.rt-r-gap-4 {
+    row-gap: var(--space-4);
+    column-gap: var(--space-4);
+  }
+  .rt-DataListRoot.rt-r-gap-5 {
+    row-gap: var(--space-5);
+    column-gap: var(--space-5);
+  }
+  .rt-DataListRoot.rt-r-gap-6 {
+    row-gap: var(--space-6);
+    column-gap: var(--space-6);
+  }
+  .rt-DataListRoot.rt-r-gap-7 {
+    row-gap: var(--space-7);
+    column-gap: var(--space-7);
+  }
+  .rt-DataListRoot.rt-r-gap-8 {
+    row-gap: var(--space-8);
+    column-gap: var(--space-8);
+  }
+  .rt-DataListRoot.rt-r-gap-9 {
+    row-gap: var(--space-9);
+    column-gap: var(--space-9);
+  }
+
+  .rt-DataListRoot.rt-r-gap-x-0 {
+    row-gap: 0px;
+  }
+  .rt-DataListRoot.rt-r-gap-x-1 {
+    row-gap: var(--space-1);
+  }
+  .rt-DataListRoot.rt-r-gap-x-2 {
+    row-gap: var(--space-2);
+  }
+  .rt-DataListRoot.rt-r-gap-x-3 {
+    row-gap: var(--space-3);
+  }
+  .rt-DataListRoot.rt-r-gap-x-4 {
+    row-gap: var(--space-4);
+  }
+  .rt-DataListRoot.rt-r-gap-x-5 {
+    row-gap: var(--space-5);
+  }
+  .rt-DataListRoot.rt-r-gap-x-6 {
+    row-gap: var(--space-6);
+  }
+  .rt-DataListRoot.rt-r-gap-x-7 {
+    row-gap: var(--space-7);
+  }
+  .rt-DataListRoot.rt-r-gap-x-8 {
+    row-gap: var(--space-8);
+  }
+  .rt-DataListRoot.rt-r-gap-x-9 {
+    row-gap: var(--space-9);
+  }
+
+  .rt-DataListRoot.rt-r-gap-y-0 {
+    column-gap: 0px;
+  }
+  .rt-DataListRoot.rt-r-gap-y-1 {
+    column-gap: var(--space-1);
+  }
+  .rt-DataListRoot.rt-r-gap-y-2 {
+    column-gap: var(--space-2);
+  }
+  .rt-DataListRoot.rt-r-gap-y-3 {
+    column-gap: var(--space-3);
+  }
+  .rt-DataListRoot.rt-r-gap-y-4 {
+    column-gap: var(--space-4);
+  }
+  .rt-DataListRoot.rt-r-gap-y-5 {
+    column-gap: var(--space-5);
+  }
+  .rt-DataListRoot.rt-r-gap-y-6 {
+    column-gap: var(--space-6);
+  }
+  .rt-DataListRoot.rt-r-gap-y-7 {
+    column-gap: var(--space-7);
+  }
+  .rt-DataListRoot.rt-r-gap-y-8 {
+    column-gap: var(--space-8);
+  }
+  .rt-DataListRoot.rt-r-gap-y-9 {
+    column-gap: var(--space-9);
+  }
+}
+
+.rt-DataListRoot {
+  margin: 0;
+  display: grid;
+  /** Value  set via props in component **/
+  grid-template-columns: var(--data-list-columns, minmax(200px, max-content) 1fr);
+}
+
+.rt-DataListItem {
+  position: relative;
+  box-sizing: border-box;
+  grid-column: span 2;
+}
+
+@breakpoints {
+  .rt-r-direction-column .rt-DataListItem {
+    display: flex;
+    flex-direction: column;
+    gap: var(--space-1);
+  }
+  .rt-r-direction-row .rt-DataListItem {
+    display: grid;
+    /** Use subgrid so all the label columns remain aligned **/
+    grid-template-columns: subgrid;
+    padding: 0;
+    gap: inherit;
+  }
+
+  .rt-DataListItem.rt-r-vaf-start {
+    align-items: start;
+  }
+
+  .rt-DataListItem.rt-r-vaf-middle {
+    align-items: center;
+  }
+
+  .rt-DataListItem.rt-r-vaf-end {
+    align-items: end;
+  }
```

Comment:
@vladmoroz [re:  your comment here](https://github.com/radix-ui/themes/pull/231#discussion_r1453790691), had to rework `align` to work with flex.